### PR TITLE
Updated Swedish translations for Companion and radio :sweden:

### PR DESCRIPTION
**Pull request by ulfhedlund**
Summary of changes: 
Updated Swedish translations for radio to make them LVGL compatible, as well as some revisions for Companion.

### CODE REVIEW

File: radio/src/translations/se.h [hunk 1]
```diff
@@ -31,6 +31,28 @@
  *  \0               -ends current string
  */
 
+
+// New strings from LVGL implementation
```
Comment (raphaelcoeffic) [inline radio/src/translations/se.h:35]: Can you please check the english original and try to have these at the same place? That eases further maintenance ;-)

**Review by raphaelcoeffic — COMMENTED** [review]
Can you please also add which radios you tested on?

**Comment by raphaelcoeffic** [comment]
@elecpower does the Companion stuff look ok? I just realised I never look at the translations.

File: radio/src/translations/se.h (hunk 1 above)
Comment (ulfhedlund) [reply radio/src/translations/se.h:35]: Tested on TX16S and Xlite.

**Comment by ulfhedlund** [comment]
> Can you please also add which radios you tested on?

Tested in simulator on TX16S and Xlite and on TX16S radio.

File: radio/src/translations/se.h (hunk 1 above)
Comment (ulfhedlund) [reply radio/src/translations/se.h:35]: Sure, I'll do that as soon as I am able.

**Comment by elecpower** [comment]
> @elecpower does the Companion stuff look ok? I just realised I never look at the translations.

Neither do I. The only real issue is the effect on gui layouts due to the length of the translated strings but that is the same for auto translated.

**Comment by ulfhedlund** [comment]
I have moved all new strings in se.h to similar places as in en.h and made some very minor adjustments to the Companion translations as well. Hopefully this can be merged now.

**Comment by ulfhedlund** [comment]
The conflict I suppose stems from me adding three new strings in the se.h file. I don't know how to resolve this but my translations are fine.

**Comment by pfeerick** [comment]
It may be cause you changed `TR_BAUDRATE` to `TR_BLUETOOTH_BAUDRATE` in addition to moving those two strings... I think if they had been moved in one commit, and the change in another it would have been fine but git obviously went and had a hissy fit now. Should be right now.

**Comment by ulfhedlund** [comment]
@pfeerick: 
Just a clarifying question - Do I need to do anything more so that this PR can be merged? If so please let me know.

**Comment by rotorman** [comment]
Tested building firmware for TX16S and Zorro (with `-DTRANSLATIONS=SE`), Companion and Simulator binaries, simulator libraries for TX16S and Zorro and creating a Windows installer. Did not see any issues (had to solve one merge conflict though, waiting for the GitHub checks to finish, then this can be merged).

**Comment by pfeerick** [comment]
> @pfeerick: Just a clarifying question - Do I need to do anything more so that this PR can be merged? If so please let me know.

No, it just needed rebasing again to account for a conflict introduced from another PR (as rebase is preferred to merge commits as merge commits pollute the commit logs unnecessarily). Anyway, it's merged now.

**Pull request merged**